Makefile

```diff
@@ -62,10 +62,11 @@ BASE_IMAGE ?= gcr.io/distroless/static:nonroot
 DOCKER_BUILDX_CMD ?= docker buildx
 IMAGE_BUILD_CMD ?= $(DOCKER_BUILDX_CMD) build
 IMAGE_BUILD_EXTRA_OPTS ?=
-IMAGE_REGISTRY ?= inftyai
+IMAGE_REGISTRY ?= $(IMAGE_REGISTRY:-inftyai)
```

Comment:
I think we should revert this, we already support your case, you may need to unset IMAGE_REGISTRY.